### PR TITLE
add carousel and btns

### PR DESCRIPTION
**Pull request by Enochen**
### Summary 

This diff adds an image carousel that appears when a user clicks a button labeled "show all photos" (the position of this button is just randomly placed as a placeholder, as we don't have a header yet to put it into).

It also adds the aforementioned button as well as a button to open up the "leave a review" modal on the Landlord page.

### Test Plan 

image

image

image

image

### CODE REVIEW

File: frontend/src/components/Review/InfoFeatures.tsx [hunk 1]
```diff
@@ -19,16 +18,14 @@ export default function InfoFeatures({
...
-          <PropertyInfo title="Property Info" info={propertyInfo} />
-          <div className={styles.divider}></div>
-          <PropertyInfo title="Property Features" info={propertyFeatures} />
-        </CardContent>
-      </Card>
-    </Grid>
+    <Card variant="outlined">
+      <CardContent>
+        <Info phone={phone} address={address} />
+        <div className={styles.divider}></div>
```
Comment (ptwu) [inline frontend/src/components/Review/InfoFeatures.tsx:24]: this is an element with no contents, so you can just do `<div ... />`

File: frontend/src/pages/LandlordPage.tsx [hunk 2]
```diff
@@ -73,70 +78,87 @@ const dummyRatingInfo: RatingInfo[] = [
 
 const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
-  const [width, setWidth] = useState(window.innerWidth);
   const [landlordData] = useState(dummyData);
   const [aveRatingInfo] = useState(dummyRatingInfo);
-  const breakpoint = 600;
+
+  const [reviewOpen, setReviewOpen] = useState(false);
```
Comment (ptwu) [inline frontend/src/pages/LandlordPage.tsx:84]: Might want to parametrize these `useState` hooks so that they don't get inferred to just be a falsy value.

**Review by ptwu — COMMENTED** [review]
Looks great! Can you resolve the merge conflict so we can see the staging site? Also left a few nits below.

File: frontend/src/pages/LandlordPage.tsx (hunk 2 above)
Comment (Enochen) [reply frontend/src/pages/LandlordPage.tsx:84]: Which hooks are you talking about? landlordData and aveRatingInfo?

File: frontend/src/components/Review/InfoFeatures.tsx (hunk 1 above)
Comment (Enochen) [reply frontend/src/components/Review/InfoFeatures.tsx:24]: ah this is not code that I wrote, I think MUI already has a <Divider /> so I'll use that instead

File: frontend/src/pages/LandlordPage.tsx (hunk 2 above)
Comment (ptwu) [reply frontend/src/pages/LandlordPage.tsx:84]: Lines 84-85: you can do `const [reviewOpen, setReviewOpen] = useState<boolean>(false);` for example

Comment (Enochen) [reply frontend/src/pages/LandlordPage.tsx:84]: Is this needed? Typescript seems to infer the type if you put "false"
image

Comment (ptwu) [reply frontend/src/pages/LandlordPage.tsx:84]: ok my bad, it only does this when you supply an empty array as the init value. disregard this

**Review by ptwu — APPROVED** [review]

**Pull request merged**